src/pg-utils.js

```diff
@@ -0,0 +1,37 @@
+const { Client } = require('pg');
+const format = require('pg-format');
+const log = require('loglevel');
+
+const env = require('./env');
+
+const runSQL = async (sql) => {
+  const pg = new Client({ connectionString: env.getPostgresUrl() });
+  await pg.connect();
+  const result = await pg.query(sql);
+  await pg.end();
```

Comment:
Please add a try / catch with a `finally` block to close the connection.

Reply:
Nice!